### ISSUE TITLE
Feature/tool request: Automatic package creation from modified game.

### ISSUE THREAD

**Issue by jpsearaphim** [sedenion/OpenModMan#30]
Since this mod manager is in conflict with other less open mod manager (ex: Steam workshop, Vertex...) it would be great if there was a way to easily port over mods from other mod managers to this mod manager.

A way the tool could achieve this: Compare a "vanilla" game root folder against a modded one. The tool hashes files on both root directories and the differing hashes would then allow to determine which files were modded and create a new package out of them. 

It would also be an easy way to make a "modpack" out of all the possibly already installed mods that someone has when moving to this mod manager.

**Comment by sedenion** [comment]
Prior to anything, this kind of feature will not be the priority unless Open Mod Manager becomes popular... Then, I vaguely understand the idea, however, is the user supposed to proceed in two step, a first scan, then a comparison + modpack ? Else, where Open Mod Manager is supposed to find the "vanilla" game root folder to compare with ?

**Comment by jpsearaphim** [comment]
Yeah, I was thinking of a two step scan with an option to browse to a root game folder for both steps. (Ask again on second step) As such; the user is free to decide if they want to start scan with a vanilla folder then add the mod and start second scan or simply have two copies of the game and scan for both.  This allows most use cases such as porting over an already modded game or simply starting fresh and porting one mod at a time.

**Comment by lmiol** [comment]
at first glance it seems like this is a big feature =)
